http_types/utils.py

```diff
@@ -56,7 +56,7 @@ def __init__(self):
         raise Exception("Do not instantiate")
 
     @staticmethod
-    def from_dict(obj: Any) -> Request:
+    def from_dict(obj: Dict) -> Request:
```

Comment:
`Dict` without types is the same as `dict` so could use that as well?

Reply:
Yes! I think using `typing.Dict` might be a better practice in general, so that it's a smaller diff once you tighten down the types of the dict. But not a big deal either way I think.